setup.cfg

```diff
@@ -2,8 +2,6 @@
 combine_as_imports=True
 force_grid_wrap=0
 include_trailing_comma=True
-known_first_party=h11, test
-known_third_party=pytest
```

Comment:
Aren't these still valid and useful?

Reply:
They're now unnecessary as any imports from the directories that isort is sorting will be considered first party. The options can be left but they will serve no function.